### PR TITLE
Feat/retire bitcoin

### PR DESCRIPTION
**Pull request by perfogic**
## Summary by CodeRabbit

- **New Features**
	- Integrated new libraries for Bitcoin bridging and SDKs.
	- Added support for a new OraiBTC Bridge network and updated Bitcoin token configurations.
	- Introduced a context for managing Bitcoin client state within the application.
	- Implemented functionality for generating Bitcoin deposit addresses and handling deposits through the new DepositBtcModalV2 component.
	- Enhanced sidebar navigation by adding a link for the new Bitcoin dashboard.
	- Launched a new Bitcoin dashboard with features for managing deposits, withdrawals, and conversions.
	- Added a "Fast Mode" toggle for improved transaction speed options.
	- Introduced new components for managing checkpoints and converting Legacy BTC to BTC V2.
	- Enhanced handling of Bitcoin transaction fees with updated logic for deposits and withdrawals.
	- Improved user interface for managing pending deposits and withdrawals.
	- Enhanced the PendingWithdraws component for better transaction handling and display options.
	- Added new component for displaying transaction outputs in a dashboard.
	- Introduced a new component for converting Legacy BTC to BTC V2.
	- Updated the Balance component to support new Bitcoin transaction handling and modal functionality.
	- Expanded functionality of the PendingWithdraws component to include additional props for better data handling.

- **Bug Fixes**
	- Resolved issues related to sidebar rendering logic for the Bitcoin dashboard.

- **Style**
	- Added new styles for the DepositBtcModalV2 component and introduced a new CSS class for fast mode.

- **Documentation**
	- Updated configuration files to reflect new dependencies and network settings.

### CODE REVIEW

**Review by quangdz1704 — DISMISSED** [review]

File: src/pages/UniversalSwap/Swap/components/SelectToken/SelectToken.tsx [hunk 1]
```diff
@@ -77,6 +77,9 @@ export default function SelectToken({
       (textChain ? item.chainId === textChain : true) &&
       (textSearch ? item.name.toLowerCase().includes(textSearch.toLowerCase()) : true)
   );
+  console.log({
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Swap/components/SelectToken/SelectToken.tsx:80]: remove log

**Review by haunv3 — APPROVED** [review]
lgtm

**Review by haunv3 — DISMISSED** [review]

**Review by haunv3 — DISMISSED** [review]
lgtm

**Review by haunv3 — APPROVED** [review]
lgtm

**Review by quangdz1704 — APPROVED** [review]

**Pull request merged**